process.tex

```diff
@@ -1,5 +1,10 @@
 \section{Process} \label{sec:process}
 
+
+This high level process has already been documented in \citeds{LDM-294}, section 3.6. 
+In order to avoid duplications, this process should be maintained only in one document.
```

Comment:
The high-level Release Management plan for  \gls{DM} is documented in \citeds{LDM-294},

Reply:
A decision has to be taken: if this section has to be maintained here, or in LDM-294 and just refer to it.
I am more inclined for the second option, remove this section from here and add in the introduction a reference to LDM-294, section 3.6.

Reply:
To me, the statement 'The high-level Release Management plan for \gls{DM} is documented in \citeds{LDM-294},' implies that LDM-294 is the main source.